pkgs/applications/altcoins/parity/beta.nix

```diff
@@ -0,0 +1,7 @@
+let
+  version     = "1.9.3";
+  sha256      = "19qyp6kafnnfhdnbq745v8zybnqizjzzc3k4701ly9hf0dvx53ka";
+  cargoSha256 = "1vdvqs6ligp5fkw5s7v44vwqwz5dqa0ipilx0piz6swz0drilima";
+  patches     = [ ./patches/vendored-sources-1.9.patch ];
```

Comment:
Maybe I should just accept #34034
Probably we could add a flag to explicitly unlock git repo support. Then we know, which packages we need to fix in case there breakage in `cargo-vendor`

Reply:
It will be cool to drop this patches, but as I see PR is WIP, is it will merged in near time?